tests/unit/amo/components/TestIcon.js

```diff
@@ -1,40 +1,38 @@
 import * as React from 'react';
-import { shallow } from 'enzyme';
 
 import Icon from 'amo/components/Icon';
+import { render } from 'tests/unit/helpers';
 
 describe(__filename, () => {
   it('maps the name to a className', () => {
-    const root = shallow(<Icon name="foo" />);
-
-    expect(root).toHaveClassName('Icon');
-    expect(root).toHaveClassName('Icon-foo');
+    const { root } = render(<Icon name="foo" />);
+    expect(root).toHaveClass('Icon');
+    expect(root).toHaveClass('Icon-foo');
   });
 
   it('allows a custom className', () => {
-    const root = shallow(<Icon name="bar" className="sup" />);
+    const { root } = render(<Icon name="bar" className="sup" />);
 
-    expect(root).toHaveClassName('Icon');
-    expect(root).toHaveClassName('Icon-bar');
-    expect(root).toHaveClassName('sup');
+    expect(root).toHaveClass('Icon');
+    expect(root).toHaveClass('Icon-bar');
+    expect(root).toHaveClass('sup');
   });
 
   it('renders alt-text as a visually hidden span', () => {
     const alt = 'Alt text!';
-    const root = shallow(<Icon alt={alt} name="bar" />);
-
-    expect(root.find('.visually-hidden')).toHaveText(alt);
+    const { getByText } = render(<Icon alt={alt} name="bar" />);
+    expect(getByText(alt)).toHaveClass('visually-hidden');
```

Comment:
does `getByAltText` work when it's visually hidden also?

Reply:
Unfortunately, because the `Icon` component doesn't render any elements that have an `alt` property, we cannot use `getByAltText` here.